snapcraft/application.py

```diff
@@ -224,6 +224,14 @@ def _run_inner(self) -> int:
             err.doc_slug = "/explanation/remote-build"
             self._emit_error(err)
             return_code = err.retcode
+        except launchpad.LaunchpadError as err:
+            self._emit_error(
+                craft_cli.errors.CraftError(
+                    f"{err}", doc_slug="/explanation/remote-build"
```

Comment:
Are doc slugs edible, or is that only [mopane worms](https://en.wikipedia.org/wiki/Gonimbrasia_belina#As_food)?